README.md

```diff
@@ -246,6 +246,64 @@ An example `<selected-recipe-string-key>` would be `"recipe_4.50_bit_mixedpalett
 
 </details>
 
+
+## <a name="using-stable-diffusion-3"></a> Using Stable Diffusion 3
+
+<details>
+  <summary> Details (Click to expand) </summary>
+
+### Model Conversion
```

Comment:
Yes great point. I'll start working on some adjustments to import diffusionkit python converters for SD3 here. This may take some time to complete if any diffusionkit updates are required, but I agree it is the right approach. Also curious your thoughts on this comment in the context of adding dependencies https://github.com/apple/ml-stable-diffusion/pull/329#discussion_r1657648657.

Reply:
I think @alejandro-isaza is better suited to discuss how to bring in dependencies on swift packages :) 
For this comment, I was just limiting to the python package dependency.